libs/utils/utils.js

```diff
@@ -563,11 +563,12 @@ function decorateHeader() {
   }
   const headerQuery = new URLSearchParams(window.location.search).get('headerqa');
   header.className = headerQuery || headerMeta || 'gnav';
+  const metadataConfig = getMetadata('breadcrumbs')?.toLowerCase()
+  || getConfig().breadcrumbs;
+  if (metadataConfig === 'off') return;
   const breadcrumbs = document.querySelector('.breadcrumbs');
-  if (breadcrumbs) {
-    header.classList.add('has-breadcrumbs');
-    header.append(breadcrumbs);
-  }
+  header.classList.add('has-breadcrumbs');
```

Comment:
Great catch, initially I assumed we would either auto create (via url) a breadcrumb or turn it off.

IMO it would lead to confusion and authoring overhead though, so we now have a config to turn auto breadcrumb generation on or off .. and another one to turn the breadcrumb one or off.

So as we have two configs, we should be checking if either the auto breadcrumbs is on or a breadcrumbs exists before we add this, to not end up with additional padding!